### PR TITLE
Remove PropTypes and DeprecatedViewPropTypes from ViewerPagerAndroid.android.js

### PR DESCRIPTION
**Pull request by ronal2do**
Related to #21342
Remove PropTypes and DeprecatedViewPropTypes from ViewerPagerAndroid.android.js

Test Plan:
----------
Flow tests succeed

- [x] yarn prettier
- [x] yarn flow-check-android
- [x] yarn flow-check-ios

Release Notes:
--------------
[GENERAL] [ENHANCEMENT] [ViewPagerAndroid.android.js] replace prop-types by Flow

### CODE REVIEW

File: Libraries/Components/ViewPager/ViewPagerAndroid.android.js [hunk 1]
```diff
@@ -72,104 +70,93 @@ export type ViewPagerScrollState = $Enum<{
...
-     * the requested page change or when user is swiping/dragging between pages)
-     * The `event.nativeEvent` object for this callback will carry following data:
-     *  - position - index of first page from the left that is currently visible
-     *  - offset - value from range [0,1) describing stage between page transitions.
-     *    Value x means that (1 - x) fraction of the page at "position" index is
-     *    visible, and x fraction of the next page is visible.
-     */
-    onPageScroll: PropTypes.func,
+type Props = $ReadOnly<{|
+  /**
```
Comment (TheSavior) [inline Libraries/Components/ViewPager/ViewPagerAndroid.android.js]: The propTypes version spreads in `DeprecatedViewPropTypes` so we probably need to include the view props here too.

```
import type {ViewProps} from 'ViewPropTypes';

type Props = $ReadOnly<{|
  ...ViewProps,
  // ...
|};
```

File: Libraries/Components/ViewPager/ViewPagerAndroid.android.js [hunk 2]
```diff
@@ -72,104 +70,93 @@ export type ViewPagerScrollState = $Enum<{
...
+class ViewPagerAndroid extends React.Component<Props> {
   componentDidMount() {
     if (this.props.initialPage != null) {
       this.setPageWithoutAnimation(this.props.initialPage);
     }
   }
 
   /* $FlowFixMe(>=0.78.0 site=react_native_android_fb) This issue was found
-   * when making Flow check .android.js files. */
+             * when making Flow check .android.js files. */
```
Comment (TheSavior) [inline Libraries/Components/ViewPager/ViewPagerAndroid.android.js]: Unintentional extra whitespace?

File: Libraries/Components/ViewPager/ViewPagerAndroid.android.js (hunk 1 above)
Comment (ronal2do) [reply Libraries/Components/ViewPager/ViewPagerAndroid.android.js]: Thanks, thanks to the feedback I'll be editing

**Comment by TheSavior** [comment]
Thanks for contributing this but I just realized when seeing that you had conflicts that ViewPager had actually already been converted in this PR which just landed: https://github.com/facebook/react-native/pull/21347. Sorry for the hassle. Maybe you could take `SwipeableQuickActions` instead?

**Comment by ronal2do** [comment]
I fixed `InputAccessoryView` and `IntegrationTestHarnessTest` instead thanks

**Pull request closed without merging**